arrow-libs/core/arrow-core-test/build.gradle.kts

```diff
@@ -0,0 +1,36 @@
+plugins {
+  alias(libs.plugins.arrowGradleConfig.multiplatform)
+  alias(libs.plugins.arrowGradleConfig.publishMultiplatform)
+}
+
+apply(from = property("ANIMALSNIFFER_MPP"))
+
+kotlin {
+  sourceSets {
+    commonMain {
+      dependencies {
+        api(projects.arrowCore)
+        api(libs.coroutines.core)
+        api(libs.kotest.assertionsCore)
+        api(libs.kotest.frameworkEngine)
+        api(libs.kotest.property)
+        implementation(libs.kotlin.stdlibCommon)
+      }
+    }
+    jvmMain {
+      dependencies {
+        implementation(libs.kotlin.stdlibJDK8)
+      }
+    }
+    jvmTest {
+      dependencies {
+        runtimeOnly(libs.kotest.runnerJUnit5)
+      }
+    }
+    jsMain {
+      dependencies {
+        implementation(libs.kotlin.stdlibJS)
+      }
+    }
```

Comment:
Is this actually needed? 🤔 

We don't rely on `stdlibJS` or `stdlibJDK8` anywhere directly, and we don't specify it for native either.

I think specifying `runnerJUnit5` is no longer needed for Kotest since M2 it resolves `framework-engine` to JVM also cc\\ @sksamuel

Reply:
I am not sure why but they are necessary, without them, some projects don't compile.

About the Kotest runner, I have no idea, just copied from the previous `build.gradle`

Reply:
Yes, it's a bit strange to me too but doesn't matter for this PR. We can make any further improvements later.

Reply:
You need junit5 for JVM because the Kotlin multi platform plugin isn't responsible for tests unlike in native and js